src/main/java/laddergame/view/InputView.java

```diff
@@ -0,0 +1,37 @@
+package laddergame.view;
+
+import laddergame.constant.ErrorMessage;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
컨벤션상 아래와 같이 명명하는게 좋아보입니다 :)  
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```